setup.py

```diff
@@ -1,6 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import unittest
+
 from setuptools import setup
 
 
+def test_suite():
```

Comment:
I tend to run tests using "python -m unittest discover" ... but this works too :)